.travis/matrix-runner.sh

```diff
@@ -19,6 +19,7 @@ if [ "${MATRIX_TYPE}" = "python" ]; then
     coverage run manage.py test --noinput --failfast --traceback --verbosity=2 --testrunner=$TESTRUNNER
 elif [ "${MATRIX_TYPE}" = "javascript" ]; then
     psql -c 'create database commcarehq' -U postgres
+    python manage.py sync_couch_views
```

Comment:
Migrate does, and I also changed it so that couch views are sync'd on pre-migrate instead of post-migrate, for exactly the reason you mentioned. However, since pre-migrate (just like post-migrate) is a per-app_label signal, it's possible though unlikely that an app gets (django-) migrated before couch views it relies on get synced (such as generic views in couchapps). This is especially true of first-time setup, since often this dependency doesn't become clear if you're keeping up with migrations as you come in, but is then a problem for people doing all the migrations at once for the first time. So, in all setup instructions I decided to leave in an explicit `python manage.py sync_couch_views` before migrate.